plugins/browser-plugin-youtube-tracking/src/buildYouTubeEvent.ts

```diff
@@ -74,7 +74,7 @@ function getMediaPlayerEntities(
 ): MediaEntities {
   const playerStates: Record<string, boolean> = {
     ended: false,
-    paused: false,
+    pause: false,
```

Comment:
Would it read better to make the change to the `YTStateEvent` enum in `constants.ts` so it can remain as `paused` here? That aligns better with the Youtube API docs:
![Screenshot 2022-01-27 at 11 28 50](https://user-images.githubusercontent.com/8820009/151350299-b4950e4b-af3f-4a5f-8961-ff950756b80d.png)